[FIX] Missing Capital (CALL) infos

capital-technologies opened the pull request:
**Project Name:** **Capital Technologies & Research**
**Website:** **https://www.mycapitalco.in**
**Description:** Capital Technologies & Research will develop a decentralized private communication system where users can easily enjoy their private communications.
**Pull Requests:**
- [X] **[152](https://github.com/jl777/coins/pull/152)@jl777/coins**
- [x] **[5](https://github.com/pbca26/agama-wallet-lib/pull/5)@pbca26/agama-wallet-lib**
- [x] **[100](https://github.com/KomodoPlatform/Agama/pull/100)@KomodoPlatform/Agama/routes/ports.js@dev**
- [x] **[72](https://github.com/KomodoPlatform/EasyDEX-GUI/pull/72)@KomodoPlatform/EasyDEX-GUI**

lukechilds commented on app/config.js:
```diff
@@ -34,6 +34,7 @@ const defaults = {
 		'SUPERNET',
 		'VTC',
 		'ZEC',
+		'CALL',
```
Can you remove this line, it shouldn't be added as a default coin.

lukechilds commented on app/renderer/assets/custom-cryptocurrency-icons/call.svg:
```diff
@@ -0,0 +1,41 @@
+<?xpacket end="w"?></metadata>
+<g id="_Group_" data-name="&lt;Group&gt;">
+    <g id="_Group_2" data-name="&lt;Group&gt;">
+      <image id="_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_Path_..." data-name="&lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; + &lt;Path&gt; +  ...
```
This isn't an SVG it's just a PNG with an SVG wrapper. We require vector graphics to support high DPI displays.

lukechilds commented on app/renderer/components/CurrencyIcon.js:
```diff
@@ -17,6 +17,7 @@ const hasCustomIcon = new Set([
 	'SUPERNET',
 	'WLC',
 	'MYTH',
+	'CALL',
```
Would you mind submitting an issue to create your icon over at https://github.com/atomiclabs/cryptocurrency-icons?

We prefer to keep all the icons over there, we just use custom assets for a few Komodo Asset Chains.

lukechilds commented on app/marketmaker/supported-currencies.js:
```diff
@@ -401,6 +401,20 @@ const supportedCurrencies = [
+		decimals: 18,
+		etomic: '0x6c28f85eb1dbc7651ce02f7af1f29890ad084b94',
+		rpcport: 80,
+	},
```
Are you able to add these alphabetically like the other currencies?

capital-technologies commented:
We have fixed all issues so far. We will submit another PR in the future with a valid SVG file. As a bonus, we have added Romanian (ro) native translation.

sindresorhus commented:
> As a bonus, we have added Romanian (ro) native translation.

We appreciate that, but that should be done in a separate PR.

capital-technologies commented:
@sindresorhus We have removed the translation files in this PR

lukechilds commented:
Thanks for the contribution @capital-technologies looking forward to those follow-up PRs 🎉
